NERODevelopment/src/controllers/signalsController.h

```diff
@@ -0,0 +1,29 @@
+#ifndef SIGNALSCONTROLLER_H
+#define SIGNALSCONTROLLER_H
+#include <QObject>
+#include <QMap>
+#include <QString>
+
+/**
+ * @brief The TheSignalsController class
+ * Controller for the signals view
+ */
+class SignalsController : public QObject {
+    Q_OBJECT
+    Q_PROPERTY(QMap<QString, QString> attributeStatus READ getAttributeStatus NOTIFY attributeStatusChanged)
+public:
+    explicit SignalsController(QObject *parent = nullptr);
```

Comment:
you also need properties for pack temp, motor temp and state of charge

Reply:
I added pointers to reference these properties from the other files. Does that work?

Reply:
where